source/particle/property/crystal_preferred_orientation.cc

```diff
@@ -642,6 +1065,45 @@ namespace aspect
                 prm.leave_subsection ();
               }
               prm.leave_subsection ();
+
+              prm.enter_subsection("D-Rex 2004");
+              {
+                prm.declare_entry ("Minerals", "Olivine: Karato 2008, Enstatite",
+                                   Patterns::List(Patterns::Anything()),
+                                   "This determines what minerals and fabrics or fabric selectors are used used for the LPO calculation. "
+                                   "The options are Olivine: A-fabric, Olivine: B-fabric, Olivine: C-fabric, Olivine: D-fabric, "
+                                   "Olivine: E-fabric, Olivine: Karato 2008 or Enstatite. The "
+                                   "Karato 2008 selector selects a fabric based on stress and water content as defined in "
+                                   "figure 4 of the Karato 2008 review paper (doi: 10.1146/annurev.earth.36.031207.124120).");
+
+                prm.declare_entry ("Mobility", "50",
+                                   Patterns::Double(0),
+                                   "The dimensionless intrinsic grain boundary mobility for both olivine and enstatite.");
+
+                prm.declare_entry ("Volume fractions minerals", "0.5, 0.5",
```

Comment:
Is 0.5,0.5 a good default value? What is the proportion in the mantle?

Reply:
You are right, given the other defaults, it have now set it to 0.7, 0.3.